.github/workflows/master.yml

```diff
@@ -137,6 +137,10 @@ jobs:
         Rscript -e 'covr::codecov()' || :
       env:
         COVR_RUNNING: true
+    - name: Test package build
+      working-directory: mlflow/R/mlflow
+      run: |
+        ./build-package.sh
```

Comment:
This step takes about 5 minutes:

https://github.com/mlflow/mlflow/runs/4577625774?check_suite_focus=true#step:16:1